daikoku/javascript/src/components/backoffice/apis/TeamApiPricing.js

```diff
@@ -504,12 +505,13 @@ export class TeamApiPricing extends Component {
       'customName',
       'customDescription',
       'allowMultipleKeys',
+      found.aggregationApiKeysSecurity ? 'aggregationApiKeysSecurity' : (this.props.tenant.aggregationApiKeysSecurity ? 'aggregationApiKeysSecurity' : undefined),
```

Comment:
seems to be painfull...a "visible" prop in bool entry woulb have done the job  〤◕‿◕〤